### PR TITLE
New queue

### PR DESCRIPTION
**Pull request by grandizzy**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -303,37 +299,17 @@ def validate_pool(pool):
         # if pool is collateralized...
         if pool.lupIndex() > ScaledPoolUtils.price_to_index_safe(pool, pool.htp()):
             # ...ensure debt is less than the size of the pool
-            assert pool.borrowerDebt <= pool.poolSize()
+            assert pool.borrowerDebt() <= pool.poolSize()
 
-        # if there are no borrowers in the pool, ensure there is no debt
-        if pool.totalBorrowers() == 0:
-            assert pool.borrowerDebt() == 0
+        # FIXME: need the pool/heap to expose borrower count to run these validations
```
Comment (EdNoepel) [inline tests/conftest.py:304]: Would be nice if we had a solution to this.  Doesn't need to be in this PR though.

File: tests/conftest.py [hunk 2]
```diff
@@ -409,7 +385,7 @@ def summarize_pool(pool):
         if pledged_collateral > 0:
             ptp = pool.borrowerDebt() * 10 ** 18 / pledged_collateral
             ptp_index = pool.priceToIndex(ptp)
-            ru = pool.depositAt(ptp_index)
+            ru = pool.depositAt(ptp_index)  # FIXME: saw this revert with under/overflow once
```
Comment (EdNoepel) [inline tests/conftest.py:388]: Unrelated to this PR.

**Review by EdNoepel — APPROVED** [review]
No issues, but as a TODO, it would be nice to have an external view which provides the total number of borrowers.

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Pull request merged**